### PR TITLE
Adding some Godot XR entries

### PR DESCRIPTION
**Pull request by BastiaanOlij**
Working on adding some FOSS Godot XR resources into this.

### CODE REVIEW

**Comment by Calinou** [comment]
@BastiaanOlij Is this ready to merge?

**Comment by BastiaanOlij** [comment]
Yeah I think we can, I'm sure I'll find more to add in the future

File: README.md [hunk 1]
```diff
@@ -136,6 +148,7 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
 
 #### Godot 4.2+
 
+- [Godot XR Template](https://github.com/godotVR/godot-xr-template) - Template for a multi level XR game
```
Comment (Calinou) [inline README.md]:
```suggestion
- [Godot XR Template](https://github.com/godotVR/godot-xr-template) - Template for a multi-level XR game.
```

File: README.md [hunk 2]
```diff
@@ -248,6 +261,7 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
 
 - [AnimatedShape2D](https://github.com/Goutte/godot-addon-animated-shape-2d) - Animate a CollisionShape2D along with the frames of an AnimatedSprite2D.
 - [Godot Google Play Game Services](https://github.com/Iakobs/godot-play-game-services) - Integrate Google Play Games Services in your Godot game.
+- [Godot XR Tools](https://github.com/godotvr/godot-xr-tools) - Basic components for XR development
```
Comment (Calinou) [inline README.md]:
```suggestion
- [Godot XR Tools](https://github.com/godotvr/godot-xr-tools) - Basic components for XR development.
```

File: README.md [hunk 3]
```diff
@@ -107,6 +108,17 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
 
 - [Cheese Hunter](https://github.com/khairul169/cheese-hunter) - A game for Godot Engine Jam of March 2016.
 
+### XR
+
+#### Godot 4.1
+
+ - [Piggy rescue](https://github.com/surreal6/piggyRescue) - Fun little example game where you chase and rescue little pigs.
```
Comment (Calinou) [inline README.md]:
```suggestion
 - [Piggy Rescue](https://github.com/surreal6/piggyRescue) - Fun little example game where you chase and rescue little pigs.
```

File: README.md [hunk 4]
```diff
@@ -107,6 +108,17 @@ See [Vivraan/godot-lang-support](https://github.com/Vivraan/godot-lang-support).
...
+
+#### Godot 3.5
+
+ - [Beep Saber VR](https://github.com/NeoSpark314/BeepSaber) - Beat saber clone in Godot.
+ - [Friday Night Funkin`](https://github.com/this-is-bennyk/Funkin-VR) - VR port of the popular rhythm game.
```
Comment (Calinou) [inline README.md]:
```suggestion
 - [Beep Saber VR](https://github.com/NeoSpark314/BeepSaber) - Beat Saber clone in Godot.
 - [Friday Night Funkin`](https://github.com/this-is-bennyk/Funkin-VR) - VR port of the popular rhythm game.
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**